blocks/mermaid/mermaid.js

```diff
@@ -0,0 +1,18 @@
+import ComponentBase from '../../scripts/component-base.js';
+// eslint-disable-next-line import/extensions
+import mermaid from './libs/mermaid.esm.min.mjs';
+
+mermaid.initialize({
+  logLevel: 'warn',
+});
+
+export default class Mermaid extends ComponentBase {
+  ready() {
```

Comment:
It seems the preview links are not working anymore. I get 404.
Not sure how many layout shifts Mermaid will create but maybe we can improve this by making `ready()` async and waiting for `mermaid.run()` now our component will be made visible only after Mermaid is initialized.

```
async ready() { 
    const code = this.querySelector('code');
    if(!code) {
      throw new Error('Cannot initialize mermaid without code content.');
    }
    code.textContent = code.textContent.replace(/:\n/g, ': \n');
    await mermaid.run({ nodes: [code] });
  }
```

Reply:
The links work again, change some configs yesterday and forgot to switch the back. I tried this but unfortunately then we seem to have a race condition as sometimes mermaid fails to initialise then with `parsing Error: svg element not in render tree`. I think for now the performance drop is fine, we can still improve things later.